pkg/logging/logging.go

```diff
@@ -113,9 +116,10 @@ func Main(argv2 string) error {
 
 // LogConfig is marshalled as "log-config.json"
 type LogConfig struct {
-	Driver string            `json:"driver"`
-	Opts   map[string]string `json:"opts,omitempty"`
-	LogURI string            `json:"-"`
+	Driver      string            `json:"driver"`
+	Opts        map[string]string `json:"opts,omitempty"`
+	HostAddress string            `json:"host"`
```

Comment:
```suggestion
	ContainerdAddress string            `json:"containerdAddress"`
```

Ideally `log-config.json` should not contain the address, but it can be refactored later

Reply:
do we need a default value for `HostAddress` ?

Reply:
already named to `adresse` so I will resolve